docs/authoring-json-rules.md

```diff
@@ -43,7 +43,7 @@ Evaluation of ARM templates is performed on the JSON representation of the templ
 
 Since most rules apply only to specific types of Azure resources, the `resourceType` property gives rule authors a shorthand to only evaluate those types of resources.  If `resourceType` is specified, the path specified in `path` becomes relative to the resource selected in the template.
 
-The behavior of the `resourceType` property is to find a property called "resources" in the current scope that is an array of objects, look for a "type" property in each of the objects, and keep only the resources where the value of "type" matches the string in `resourceType`.  See [Scopes](#scopes) for more information on scopes.
+The behavior of the `resourceType` property is to find a property called "resources" in the current scope that is an array of objects, look for a "type" property in each of the objects, and keep only the resources where the full value of "type" matches the string in `resourceType`. Consider that if a resource of type `Microsoft.Sql/servers` has a child of type `auditingSettings`, then the full type of the child is actually `Microsoft.Sql/servers/auditingSettings`. See [Scopes](#scopes) for more information on scopes.
```

Comment:
Update to say that resources can also be matched within children of child resources at the current scope (you may need to significantly reword this statement into a few sentences).